utils/build.js

```diff
@@ -166,7 +166,7 @@ function generateBuildConfigs({formatToPath, baseDir, moduleName,
     buildConfigs.push({
       rollupConfig: {
         entry: entry || path.join(baseDir, 'src', 'index.js'),
-        plugins: [devReplacePlugin, ...basePlugins, babelPlugin],
```

Comment:
There's a back-and-forth about minification in the dev build in https://github.com/GoogleChrome/workbox/pull/594#issuecomment-305903950